.gitignore

```diff
@@ -8,7 +8,6 @@ manifest.json
 test/credentials.json
 test/binaries
 *.swp
-.idea
```

Comment:
There are more IDE specific ignore lines... Could you compare to the hsldevcom version and reset to their state?